__tests__/index.test.js

```diff
@@ -100,18 +100,64 @@ describe("GET /api/articles", () => {
         });
         expect(allArticles.length).toBe(13);
         allArticles.forEach((article) => {
-        const expectedCreatedAt = new Date('2020-01-15T22:21:00.000Z');
           expect(article).toMatchObject({
-              article_id: expect.any(Number),
-              title: expect.any(String),
-              topic: expect.any(String),
-              author: expect.any(String),
-              created_at: expect.any(String),
-              votes: expect.any(Number),
-              article_img_url: expect.any(String),
-              comment_count: expect.any(String) 
-          })
+            article_id: expect.any(Number),
+            title: expect.any(String),
+            topic: expect.any(String),
+            author: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String),
+            comment_count: expect.any(String),
+          });
         });
       });
   });
 });
+describe("GET /api/articles/:article_id/comments", () => {
+  test("return a status code of 200 and response body with an array of all comments from a particular article", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then((response) => {
+        const allComments = response.body.comments;
+        expect(allComments.length).toBe(11);
+        allComments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            author: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            body: expect.any(String),
+            article_id: expect.any(Number),
```

Comment:
you need to check the article id is the one you've made the request to - in this case 1. 

Otherwise, you cant be sure it linked to the correct article.